extensions/wikia/Email/Controller/FounderDigestController.class.php

```diff
@@ -0,0 +1,307 @@
+			throw new Check( 'Invalid value passed for `newUsers`' );
+		}
+	}
+
+	/**
+	 * Asserts that the `pageEdits` value is passed
+	 *
+	 * @throws \Email\Check
+	 */
+	protected function assertPageEditsSet() {
+		if ( empty( $this->pageEdits ) && $this->pageEdits !== '0' ) {
+			throw new Check( 'Invalid value passed for `pageEdits`' );
+		}
+	}
+
+	/**
+	 * Returns list of details for the digest
+	 *
+	 * @return array
+	 */
+	protected function getDetailsList() {
+		return [
+			[
+				'iconSrc' => \GlobalFile::newFromText( 'Page-views.png', \Wikia::NEWSLETTER_WIKI_ID )->getUrlGenerator()->url(),
+				'detailsHeader' => $this->getMessage( 'emailext-founder-digest-views-header', $this->pageViews )->parse(),
+				'details' => $this->getMessage( 'emailext-founder-digest-views-description-1' )->text()
+			],
+			[
+				'iconSrc' => \GlobalFile::newFromText( 'Number-of-edits.png', \Wikia::NEWSLETTER_WIKI_ID )->getUrlGenerator()->url(),
+				'detailsHeader' => $this->getMessage( 'emailext-founder-digest-edits-header', $this->pageEdits )->parse(),
+				'details' => $this->getMessage( 'emailext-founder-digest-edits-description' )->text()
+			],
+			[
+				'iconSrc' => \GlobalFile::newFromText( 'New-users.png', \Wikia::NEWSLETTER_WIKI_ID )->getUrlGenerator()->url(),
+				'detailsHeader' => $this->getMessage( 'emailext-founder-digest-users-header', $this->newUsers )->parse(),
+				'details' => $this->getMessage( 'emailext-founder-digest-users-description' )->text()
+			] 
+		];
+	}
+
+	public function getButtonText() {
+		return $this->getMessage( 'emailext-founder-activity-digest-link-label' )->text();
+	}
+
+	public function getButtonLink() {
+		return \GlobalTitle::newFromText( 'WikiActivity', NS_SPECIAL, $this->wikiId )->getFullURL();
+	}
+
+	protected function getCommunityFooterMessage() {
+		return $this->getMessage(
+			'emailext-founder-activity-digest-footer',
+			\GlobalTitle::newMainPage( \Wikia::COMMUNITY_WIKI_ID )->getFullURL()
+		)->parse();
+	}
+
+	protected static function getEmailSpecificFormFields() {
+		$formFields = [
+			'inputs' => [
+				[
+					'type' => 'text',
+					'name' => 'newUsers',
+					'label' => 'New users',
+					'tooltip' => 'Number of new visitors to the wiki'
+				],
+				[
+					'type' => 'text',
+					'name' => 'pageEdits',
+					'label' => 'Page edits',
+					'tooltip' => 'Number of edits made to the wiki by users'
+				]
+			]
+		];
+
+		return array_merge_recursive( $formFields, parent::getEmailSpecificFormFields() );
+	}
+}
+
+class FounderPageViewsDigestController extends FounderDigestController {
+	const TRACKING_CATEGORY_EN = TrackingCategories::FOUNDER_VIEWS_DIGEST_EN;
+	const TRACKING_CATEGORY_INT = TrackingCategories::FOUNDER_VIEWS_DIGEST_INT;
+
+	public function initEmail() {
+		$this->wikiId = $this->request->getVal( 'wikiId' );
+		$this->pageViews = $this->request->getVal( 'pageViews' );
+		$this->assertValidParams();
+
+		$this->pageViews = number_format( $this->pageViews );
```

Comment:
We should use the facilities in the ```Language``` class rather than rely on PHP's ```number_format``` (see  https://one.wikia-inc.com/wiki/Internationalization/Technical#Numbers)

You can use ```\Language::factory( $this->targetLang );``` to get the right language object.